shell/src/main/java/org/apache/accumulo/shell/commands/GrepCommand.java

```diff
@@ -71,10 +71,11 @@ public int execute(final String fullCommand, final CommandLine cl, final Shell s
       scanner.setTimeout(getTimeout(cl), TimeUnit.MILLISECONDS);
 
       setupSampling(tableName, cl, shellState, scanner);
+      addScanIterators(shellState, cl, scanner, "");
 
       for (int i = 0; i < cl.getArgs().length; i++) {
         setUpIterator(Integer.MAX_VALUE - cl.getArgs().length + i, "grep" + i, cl.getArgs()[i],
-            scanner, cl, negate);
+            scanner, cl, negate, shellState);
```

Comment:
`shellState` is newly passed, but it seems like its not used.

Reply:
correct, removed shellState from method signature